test/cli/repl/t20-lexer/expected

```diff
@@ -30,7 +30,10 @@ Check error:
       mean                 : (Num b) => Element a b -> Aggregate a Double
       date                 : Element Definitely DateTime
       min                  : Element a b -> Aggregate a b
-      sd                   : (Num b) => Element a b -> Aggregate a Double
+      sd                   : (Element =: ReturnOfLet Pure t$107
+                             ,t$105 =: ReturnOfLet a a
+                             ,t$106 =: ReturnOfLet a t$105
+                             ,t$107 =: ReturnOfLet a t$106) => a b Double -> Aggregate b Double
```

Comment:
oh ok that's not related to your typechecker changes, just the different definition of sd. I'll look at that later.